x-pack/plugin/esql/qa/testFixtures/src/main/java/org/elasticsearch/xpack/esql/CsvTestUtils.java

```diff
@@ -354,7 +354,8 @@ public static ExpectedResults loadCsvSpecValues(String csv) {
                 for (int i = 0; i < row.size(); i++) {
                     String value = row.get(i);
                     if (value == null) {
-                        rowValues.add(null);
+                        // Empty cells are converted to null by SuperCSV. We convert them back to empty strings.
+                        rowValues.add("");
```

Comment:
SuperCSV is causing us some trouble here. Was it turning the `""` in the string.csv-spec into `null` because it's the last line?

Reply:
All empty cells, whether quoted or unquoted (`a||b`), are returned as null from SuperCSV. In any row.
For now, this fix makes the trick. It's not very intuitive, but it works, and doesn't affect other tests